snuba/datasets/schemas/tables.py

```diff
@@ -183,7 +197,7 @@ def __init__(
         storage_set_key: StorageSetKey,
         mandatory_conditions: Optional[Sequence[MandatoryCondition]] = None,
         prewhere_candidates: Optional[Sequence[str]] = None,
-        order_by: str,
+        order_by: Sequence[Expression],
```

Comment:
I did not change partition_by and sample_expr for two reasons:
- This whole part of the schema is not going to stay here for long since it is being replaced by the new migration system. This is already not used anymore where we rolled out migrations
- reduce the size of this PR. It would have been twice as big.